lib/melodica_inventory_web/router.ex

```diff
@@ -48,6 +48,7 @@ defmodule MelodicaInventoryWeb.Router do
     resources "/categories", CategoryController, only: [:edit, :update, :delete, :new, :create]
     resources "/variations", VariationController, only: [:edit, :update, :delete, :new, :create]
     resources "/items", ItemController, only: [:edit, :update, :delete]
+    put "/items/:id/delete_images", ItemController, :delete_images
```

Comment:
May be we should stick to REST here? Something like `resources "/item_images/:item_id", ItemImagesController, only: [:delete]`